### PR TITLE
Release Notes for Kubernetes v1.19.0

### PR DESCRIPTION
**Pull request by puerco**
/kind documentation

#### What this PR does / why we need it:

Final release notes document for Kubernetes v1.19.0 

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
/assign @saschagrunert  @onlydole 
/cc @JamesLaverack @reylejano-rxm @wilsonehusin @AGMETEOR 

Signed-off-by: Adolfo García Veytia (Puerco) <adolfo.garcia@uservers.net>

### CODE REVIEW

File: releases/release-1.19/release-notes-draft.md [hunk 1]
```diff
@@ -1,7 +1,122 @@
...
+
+### Production images moved to community control
+
+As of Kuberenetes v1.19, Kubernetes container images are stored on a community-controlled storage bucket, 
+located at `{asia,eu,us}.gcr.io/k8s-artifacts-prod`. The `k8s.gcr.io` vanity domain has been updated 
+to this new bucket. This brings production artefacts under community control.
+
+### KubeSchedulerConfiguration graduates to Beta
+
+SIG Scheduling graduates `KubeSchedulerConfiguration` to Beta. The [KubeSchedulerConfiguration](https://deploy-preview-20785--kubernetes-io-master-staging.netlify.app/docs/reference/scheduling/config) feature allows you to tune the algorithms and other settings of the kube-scheduler. You can easily enable or disable specific functionality (contained in plugins) in selected scheduling phases without having to rewrite the rest of the configuration. Furthermore, a single kube-scheduler instance can serve different configurations, called profiles. Pods can select the profile they want to be scheduled under via the `.spec.schedulerName` field.
```
Comment (wilsonehusin) [inline releases/release-1.19/release-notes-draft.md:67]: the URL referenced here is still pointing to staging — is it intentional?

**Review by JamesLaverack — APPROVED** [review]
/lgtm

File: releases/release-1.19/release-notes-draft.md (hunk 1 above)
Comment (saschagrunert) [reply releases/release-1.19/release-notes-draft.md:67]: Nope, this needs a fix :+1:

Comment (JamesLaverack) [reply releases/release-1.19/release-notes-draft.md:67]: Nice catch. :+1:

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by JamesLaverack** [comment]
🎉 
/honk

File: releases/release-1.19/release-notes-draft.md (hunk 1 above)
Comment (puerco) [reply releases/release-1.19/release-notes-draft.md]: Fixed, the final URL 404s now but should work after the website freeze is lifted.
Thanks for the catch @wilsonehusin  !

**Comment by puerco** [comment]
:tada:  :partying_face: Thanks everyone!
/pony

**Pull request merged**